app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class Api::V1::UsersController < Api::V1::BaseController
+  before_action :set_user, only: [:show, :update]
+  before_action :authenticate_user!
+  before_action :check_access, only: [:update]
+
+  def index
+    @users = paginate(User.all)
+    @options = {}
+    @options[:meta] = meta_attributes(@users)
+    render json: Api::V1::UsersSerializer.new(@users, @options)
+  end
+
+  def show
+    render json: Api::V1::UserSerializer.new(@user)
```

Comment:
Don't expose full details here, just enough details to show profile is enough

Reply:
The attributes being serialized are:
`attributes :id, :email, :name, :admin,
             :country, :educational_institute,
             :subscribed, :created_at, :updated_at
`
These are just enough for the profile.. What should i remove from here ?

Reply:
We don't want show users email in the public profile, it can cause spamming in future